lib/dolly/document.rb

```diff
@@ -19,6 +19,7 @@ def update_properties options = {}
       options.each do |property, value|
         send(:"#{property}=", value)
       end
+      updated_at = DateTime.now if has_updated_at?
```

Comment:
I dont want to hardcode this as DateTime and would rather take the class of the property.  Going to do that in the next commit